printer/printer.go

```diff
@@ -43,14 +43,14 @@ func SetSingle(single bool) {
 // template
 func PrintT(templateString string, v interface{}) {
 	switch printer.Format {
-	case FORMAT_PLAIN:
+	case FormatPlain:
 		t := template.Must(template.New("").Parse(templateString))
 		var tpl bytes.Buffer
-		t.Execute(&tpl, v)
+		_ = t.Execute(&tpl, v)
```

Comment:
Yes, I was thinking what to do here and is hard to decide what is the best way so I left that way by now. 

Giving it a second though could be useful to catch that error and call the `PrintError` in case we have an error parsing the print template. What do you think?

Reply:
:+1: for just calling `PrintError`.